src/main/java/org/minions/devfund/bruno/html/HtmlReader.java

```diff
@@ -0,0 +1,29 @@
+package org.minions.devfund.bruno.html;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Queue;
+
+/*
+ * SD2x Homework #2
+ * This class contains a method that will read an HTML file and convert it to a Queue of HtmlTags.
+ * It is simply provided as a convenience class for you to use during your testing.
+ * You do not need to submit this code.
+ */
+
+public class HtmlReader {
+
+    public static Queue<HtmlTag> getTagsFromHtmlFile(String filename) throws IOException {
+        InputStream stream = new FileInputStream(filename);
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Use try-with-resources or close this "FileInputStream" in a "finally" clause. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2095)